app/models/service_template_ansible_playbook.rb

```diff
@@ -93,6 +93,7 @@ def self.build_parameter_list(name, description, info)
       :playbook                 => playbook.name,
       :inventory                => tower.provider.default_inventory,
       :become_enabled           => info[:become_enabled].present?,
+      :verbosity                => info[:verbosity] || 0,
```

Comment:
@bzwei - Yes default is 0.  If that param is not passed in then Tower sets it to 0.  If it is passed it needs to be an integer between 0 and 5.